pkg/ingress/query.go

```diff
@@ -49,7 +49,7 @@ func (i *QueryIngress) Get() *extv1beta1.Ingress {
 	return &extv1beta1.Ingress{
 		TypeMeta: metav1.TypeMeta{
 			Kind:       "Ingress",
-			APIVersion: "extensions/v1beta1",
+			APIVersion: "networking/v1beta1",
```

Comment:
I think this should be `networking.k8s.io/v1beta1`, as shown in https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource.